pyproject.toml

```diff
@@ -63,11 +63,12 @@ dependencies = [
     "GitPython ~=3.1",
     "hatch ~=1.12",
     "pytest ~=8.3",
+    "pytest-xdist ~=3.6",
     "pre-commit"
 ]
 
 [[tool.hatch.envs.test.matrix]]
 python = ["3.10", "3.11", "3.12", "3.13"]
 
 [tool.hatch.envs.test.scripts]
-run = "pytest {args}"
+run = "pytest {args:--numprocesses=auto}"
```

Comment:
Was this change intentional?  I've often had cases where I couldn't use pytest-xdist.

Reply:
It was, as I was waiting longer than desired for the tests to finish. To be honest, it's besides the point of the issue, so I should have made it separate.

Reply:
Okay, I've removed those changes here and made a separate PR https://github.com/pyOpenSci/pyos-package-template/pull/55